common/factorization/par_ilut_select_kernels.hpp.inc

```diff
@@ -33,15 +33,17 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 namespace kernel {
 
 
-constexpr auto searchtree_width = 1 << sampleselect_searchtree_height;
-constexpr auto searchtree_inner_size = searchtree_width - 1;
-constexpr auto searchtree_size = searchtree_width + searchtree_inner_size;
+static constexpr auto searchtree_width = 1 << sampleselect_searchtree_height;
+static constexpr auto searchtree_inner_size = searchtree_width - 1;
```

Comment:
I also did some changes on these from dpcpp matrix review.
use `constexpr int` not `constexpr auto` if it is initialized by value.
Could you also do it here?